bindata/network/ovn-kubernetes/sdn-public.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ConfigMap
+apiVersion: v1
+immutable: true
+metadata:
+  name: sdn-public
```

Comment:
To us, SDN means Openshift-SDN. It may confuse us and uses to name the configmap this name.